jinja_utils.py

```diff
@@ -59,17 +59,6 @@ def get_jinja_env(dir_path):
         os.path.dirname(__file__), dir_path))
     env = jinja2.Environment(autoescape=True, loader=loader)
 
-    def include_js_file(filepath):
-        """Include a raw JS file in the template without evaluating it."""
-        assert filepath.endswith('.js')
-        raw_file_contents = loader.get_source(env, filepath)[0]
-        if filepath == _OPPIA_MODULE_DEFINITION_FILE:
-            return jinja2.Markup(raw_file_contents)
-        else:
-            # Wrap the file in an immediately-invoked function expression
-            # (IIFE) to prevent pollution of the global scope.
-            return jinja2.Markup('(function() {%s})();' % raw_file_contents)
```

Comment:
Well, it's kind of the same issue as importing a third-party lib using "script src". We don't IIFE that either...

The options don't look great: http://stackoverflow.com/questions/9635973/can-i-sandbox-3rd-party-javascript-to-protect-the-global-namespace. Perhaps this means we should just include IIFEs in our JS files.